core/corehttp/gateway_handler.go

```diff
@@ -510,7 +506,6 @@ func (i *gatewayHandler) putHandler(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 	default:
-		log.Warningf("putHandler: unhandled resolve error %T", ev)
```

Comment:
If this is a user error, we shouldn't be returning a 502. If it is, we should log it.

Reply:
Mh yes indeed - I'll cover this in the upcoming gateway+coreapi PR.